compiler-rt/lib/radsan/radsan_context.cpp

```diff
@@ -46,9 +44,7 @@ void Context::expectNotRealtime(const char *intercepted_function_name) {
   if (inRealtimeContext() && !isBypassed()) {
     bypassPush();
     printDiagnostics(intercepted_function_name);
-    if (get_error_action_() == OnErrorAction::ExitWithFailure) {
-      exit(EXIT_FAILURE);
-    }
+    exit(EXIT_FAILURE);
```

Comment:
We sure we don't want to have an `enum class` option instead for the time being, or are you keen to minimise cognitive load for v0 upstreaming?

Reply:
I'm open to it if you feel strongly about it. I don't think it's hard to add it back in when we add the functionality back in the future, so I'd say I personally lean towards how it is. This leaning is very slight though, I defer to you. This is the cleanest total rip-out I could do (which I think has value, as opposed to a partial)